engine/installation/linux/docker-ce/cruxlinux.md

```diff
@@ -3,6 +3,7 @@ description: Docker installation on CRUX Linux.
 keywords: crux linux, Docker, documentation,  installation
 redirect_from:
 - /engine/installation/cruxlinux/
+- /engine/installation/linux/docker-ce/cruxlinux/
```

Comment:
We can remove this page altogether and redirect to the install landing page. This page describes installing from packages we don't maintain (we never shipped packages for this distro)